### DISCUSSION TITLE
Circle Size in Lazer Editor has inverted Values?

### DISCUSSION THREAD

**Discussion by DiogoSL** [ppy/osu#23993] [Q&A] [answered]
Is it nomal that the circle size when increased in the new editor, actually decreases the circle size?
And the opposite happens when decreasing the circle size.
![issue_circle_size](https://github.com/ppy/osu/assets/29704333/2ed87861-2243-40c5-98a6-3e2104179f48)

**Comment by monochrome22** [comment, marked as answer]
That's normal. It's meant to show level of difficulty, 0 being easiest and 10 being the most difficult. Same applies to other stats. Also, it works the same way in Stable, so it's not about "new Lazer editor".

**Reply by DiogoSL** [reply]
Sorry for the confusion and thanks for the explanation ^^
